client/core/core.go

```diff
@@ -131,6 +131,9 @@ func New(cfg *Config) (*Core, error) {
 		// Allowing to change the constructor makes testing a lot easier.
 		wsConstructor: comms.NewWsConn,
 	}
+	// Populate the initial user data. User won't include any DEX info yet, as
+	// those are retrieved when Run is called and the core connects to the DEXes.
+	core.refreshUser()
```

Comment:
Can we do all the wallet loading too?  i.e.  https://github.com/decred/dcrdex/blob/02ca2ae833725bab76a496015ba4d18b64ba2e2d/client/core/core.go#L921-L938

And limit the Run/initialize work to dex connections?

Reply:
But even with that, it's not clear to me when it is OK to use `Core.Login` since it's hitting `c.conns`.  

Interestingly in the hack app work, it wasn't Login that was having trouble (it did connect to the dex configured on the previous run), it was `OpenWallet` that failed presumably because `initialize` hadn't reached `refreshUser`? But apparently it had connected already.